### ISSUE TITLE
Design breakdown

### ISSUE THREAD

**Issue by ZorckMaster** [hzi-braunschweig/SORMAS-Angular#39]
Break down the current design in smaller tasks to be able to start implementing them.

**Comment by ZorckMaster** [comment]
Added a docx file with some tasks for the design breakdown.
It is not a final version so comments, modification requests are welcome.

[Sormas_design_breakdown.docx](https://github.com/hzi-braunschweig/SORMAS-Angular/files/5942020/Sormas_design_breakdown.docx)

**Comment by ZorckMaster** [comment]
Please consider that all the tasks are based on dummy data until we can't use real data for any reason.

**Comment by laurentiustanvitagroup** [comment]
The tasks suggested look great. I've grouped them under epics / stories as to facilitate and give more transparency

[Sormas Angular Upcoming backlog items.pdf](https://github.com/hzi-braunschweig/SORMAS-Angular/files/5942879/Sormas.Angular.Upcoming.backlog.items.pdf)

**Comment by alexandrudanpop** [comment]
> Please consider that all the tasks are based on dummy data until we can't use real data for any reason.

Hey @ZorckMaster, thanks for the document. Will also look into it. I just want to mention that we can use real data that is filled in the actual app - https://test-de1.sormas.netzlink.com/sormas-ui/ . This data should be available from the APIs.

**Comment by ZorckMaster** [comment]
@alexandrudanpop Yes, I we can, but would like to mention that from the breakdown's perspective it is not the most important thing where the data is coming from that's why I said Dummy or from API. So we can do it either way, just need to have the elements of the design ready, then we can still focus on how we get the data. Of course we can directly use data coming from the API, no problem. I thought this is a secondary problem. If not, I can correct the doc where I said dummy data, it can be data from API.

**Comment by alexandrudanpop** [comment]
@ZorckMaster yep the data should be from API always - we have already the API client and all the fields should be mapped from the API responses. This adds of course extra complexity that we need to factor in.
